content/python/concepts/built-in-functions/terms/issubclass/issubclass.md

```diff
@@ -0,0 +1,49 @@
+## **issubclass()**
+used to check if a class is a subclass of another class or not.
+
+### Syntax
+`issubclass()` take two parameters:
+
+`issubclass(class, classinfo)`
+- **class** - class to be checked
+- **classinfo** - class, type, or tuple of classes and types 
+
+**Return type:** `True` if object is subclass of a class, or any element of the tuple, otherwise `False`.
+
+
```

Comment:
```suggestion

The `issubclass()` function take two parameters:

- `class`- a validly defined class
- `candidateClasses` - a single class, type, or [tuple](https://www.codecademy.com/resources/docs/python/tuples) of classes and types

If any relationship exists, `True` is returned, otherwise `False`.

```

Reply:
@cacaprog Looks like this suggestion remains from the last bit of feedback. Can we implement this?